core/mempool/src/pool.rs

```diff
@@ -195,18 +206,32 @@ impl PriorityPool {
                 *reduce_len += 1;
             }
         }
+
+        let timeout = if number > self.timeout_config {
```

Comment:
How to gurantee `timeout_gap` doesn't contain keys less than `number - self.timeout_config`  ?

Reply:
the number is block number, each block will call this function on commit to DB, it is a monotonically increasing number